test/modules/post/test/extapi.rb

```diff
@@ -30,8 +30,8 @@ def setup
       vprint_status("Loading extapi extension...")
       begin
         session.core.use("extapi")
-      rescue Errno::ENOENT
-        print_error("This module is only available in a windows meterpreter session.")
+      rescue Errno::ENOENT, Rex::Post::Meterpreter::ExtensionLoadError
+        print_status("This module is only available in a windows meterpreter session.")
```

Comment:
Does this error message still make sense for the additional exception you're handling here?
would there be any benefit to printing out which exception it was?